### ISSUE TITLE
if object query is random, as shown in code, how to evaluate to get a steady result? 

### ISSUE THREAD

**Issue by RedBlack888** [facebookresearch/detr#597]
if object query is random, as shown in code, how to evaluate to get a steady result when inference? 
`        self.query_embed = nn.Embedding(num_queries, hidden_dim)
`
       
` hs = self.transformer(self.input_proj(src), mask, self.query_embed.weight, pos[-1])[0]`

Best.

**Comment by AjibolaPy** [comment]
It's learnable. I think you'll use the learnt weight on inference
